packages/@okta/vuepress-site/docs/guides/index.md

```diff
@@ -38,7 +38,7 @@ guides:
  - build-sso-integration
  - custom-sms-messaging
  - submit-app
- - production-deployment
+ - deployment-checklist
```

Comment:
There are 2 broken links in this file (line 137, 138):
 `* [Pre-launch checklist](/docs/guides/production-deployment/deployment-checklist/)`
` * [Deploy your app](/docs/guides/deploy-your-app/overview/)`

Reply:
fixed.